device/mellanox/x86_64-nvidia_sn4280-r0/Mellanox-SN4280-O8C80/sai_4280_8x400g_80x100g.xml

```diff
@@ -0,0 +1,266 @@
+<?xml version="1.0"?>
```

Comment:
Can you please check this file? It appears this is not updated according to latest spec